DiscordLink/Source/Data/DiscordChannelNameTypeConverter.cs

```diff
@@ -69,6 +74,7 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
         {
             if (value != null && destinationType == typeof(string) && value.GetType() == typeof(ulong))
             {
+                if (DiscordLink.Obj?.Client?.Guild?.Channels?.Values == null) return "No Connection";
```

Comment:
Should have pointed this out in the previous merge, but the style I've used is to not do one line ifs.
Could you swap these to this style?
if()
   return